implementations/rust/ockam/ockam_core/src/routing/message/local_message.rs

```diff
@@ -0,0 +1,38 @@
+use crate::{lib::Vec, TransportMessage};
+use serde::{Deserialize, Serialize};
+
+/// LocalMessage. Consists of TransportMessage + some local info added by Workers within the same node
+#[derive(Serialize, Deserialize, Debug, Clone, Hash, Ord, PartialOrd, Eq, PartialEq)]
+pub struct LocalMessage {
```

Comment:
In the PR description, could you briefly describe the need for `LocalMessage` and `RouterMessage` and how it relates to Trust Policy implementation?

Reply:
For LocalMessage definitely.
RouterMessage was just moved to a different location without changes, so it has nothing to do with TrustPolicy. I also think it has pretty good doc-comment.